cloudformation/dynamoDb.yaml

```diff
@@ -220,6 +220,30 @@ Resources:
           - referrals
     DeletionPolicy: Retain
 
+  CampaignReferralsV2DBTable:
```

Comment:
Is there a way we can name this so it's agnostic of the temporary api versioning we put in place (as I plan to remove this at a later date)?